openvino_xai/explainer/explanation.py

```diff
@@ -132,21 +132,61 @@ def _select_target_indices(
                 raise ValueError("Provided targer index {targer_index} is not available among saliency maps.")
         return target_indices
 
-    def save(self, dir_path: Path | str, name: str | None = None) -> None:
-        """Dumps saliency map."""
+    def save(
+        self,
+        dir_path: Path | str,
+        prefix_name: str | None = "",
+        suffix_name: str | None = "target",
+        postfix_name: str | None = "",
```

Comment:
Is it usual to put "suffix" and "postfix" together?
I thought they are rather same things. Anyway, isn't it a little bit confusing to users?

Reply:
`Suffix_name` goes before inserted label_name, while the `postfix` - after. I will add the doc comment like this.

        For the name 'image_name_target_aeroplane.jpg': prefix = 'image_name', suffix = 'target', label name = 'aeroplane', postfix = ''.

Theoretically, we can combine `prefix` and `suffix` since they will be combined and put before `label_name`, but they have different meanings. `Suffix` -  to name the `target` part, where we will highlight the label name of saliency map.

I'm looking for a way to make it more clear

Reply:
What about
prefix_name -> file_prefix
suffix_name -> target_prefix
postfix_name -> target_suffix